addons/point_of_sale/models/pos_order.py

```diff
@@ -334,7 +334,7 @@ def _default_pricelist(self):
     date_order = fields.Datetime(string='Order Date', readonly=True, index=True, default=fields.Datetime.now)
     user_id = fields.Many2one('res.users', string='Salesman', help="Person who uses the cash register. It can be a reliever, a student or an interim employee.", default=lambda self: self.env.uid)
     amount_tax = fields.Float(compute='_compute_amount_all', string='Taxes', digits=0)
-    amount_total = fields.Float(compute='_compute_amount_all', string='Total', digits=0)
+    amount_total = fields.Float(compute='_compute_amount_all', string='Total', digits=0, store=True)
```

Comment:
If store, check triggers are correctly set on compute method. It does not seem to be the case.